swiftpkg/internal/resource_files.bzl

```diff
@@ -6,23 +6,24 @@ load("@bazel_skylib//lib:sets.bzl", "sets")
 # For a list of the auto-discovered resource types, see
 # https://github.com/apple/swift-package-manager/blob/main/Sources/PackageLoading/TargetSourcesBuilder.swift#L634-L677
 _XIB_EXTS = ["nib", "xib", "storyboard"]
-_ASSET_CATALOG_EXTS = ["xcassets"]
```

Comment:
Why was this removed?

Reply:
It's handled by `_is_under_asset_catalog_dir` for actual files under it, and we don't want the directories _themselves_ to ever be valid resources.